TombEngine/Game/Lara/lara_fire.cpp

```diff
@@ -832,12 +837,25 @@ FireWeaponType FireWeapon(LaraWeaponType weaponType, ItemInfo* targetEntity, Ite
 	}
 	else
 	{
-		Statistics.Game.AmmoHits++;
-
 		target = origin + (directionNorm * bestDistance);
 
 		auto vTarget = GameVector(target);
 
+		if (targetEntity->ObjectNumber == ID_PUNA_BOSS)
```

Comment:
We are trying to get rid of such hardcoded hacks. Please find another way to do this.

Reply:
This one is fixed.